packages/client-sdk-web/src/internal/control-client.ts

```diff
@@ -36,6 +40,7 @@ export class ControlClient<
 > implements IControlClient
 {
   private readonly clientWrapper: control.ScsControlClient;
+  private readonly clientAuthWrapper: auth.AuthClient;
```

Comment:
ditto about the clients

Reply:
https://github.com/momentohq/client-sdk-nodejs/pull/393#discussion_r1160137539